### DISCUSSION TITLE
[Mania] Opinions on Percy LN not working in lazer

### DISCUSSION THREAD

**Discussion by axeties** [ppy/osu#26294] [Q&A] [answered]
Just looking for different opinions on Percy LN not working, personally i think it's rubbish that LN tails that don't misrepresent the length of sliders are purposefully scrapped.
I get that some older players argue that it decreases skill requirement, but i think since the meta is based around percy that the devs should at least have some community vote on it.

**Comment by bdach** [comment]
we made it work though (https://github.com/ppy/osu/pull/22389). not in the exact same way, but it *should* work. if it doesn't on your skin then please link skin.

> personally i think it's rubbish that LN tails that don't misrepresent the length of sliders are purposefully scrapped

this is really weird though. how does not putting 20000px sprites in your skin "misrepresent the length of sliders"?

**Reply by Adrriii** [reply]
What he's probably referring to, is that current LN tail implementation can misrepresent the release timing on some skin varieties. On Bar skins, the issue is not really present, however on other skin types, it may be preferable for the LN Tail to stop at the timing, not start from it.

This would increase readability without any sort of hack.
Some people may still prefer to actually cut their LNs (percy), but it feels like a harder thing to support.

Current LN tail:
![image](https://github.com/ppy/osu/assets/16978299/b45d7b29-32bb-40d6-9d2a-767e57a870f2)

Desired LN tail:
![image](https://github.com/ppy/osu/assets/16978299/623ebb01-f798-4acf-8b29-232f3ceedbc6)

**Comment by peppy** [comment, marked as answer]
It will be reimplemented in a manageable way. We've done our best to support textures a GPU cannot load. You're welcome to improve.

**Comment by Robiimadot** [comment]
I and the mania community would be really happy for the percy support. thank for you work and take breaks too.


![osu_2024-02-10_21-51-24](https://github.com/ppy/osu/assets/65800580/73ff5b48-1d70-4e14-a2a7-f6bb684b1025)
